streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/ExecutionGraph.java

```diff
@@ -19,31 +18,31 @@
 public class ExecutionGraph implements Serializable {
   private long buildTime;
   private List<ExecutionNode> executionNodeList;
-  private List<RayActor<JobWorker>> sourceWorkers = new ArrayList<>();
-  private List<RayActor<JobWorker>> sinkWorkers = new ArrayList<>();
+  private List<RayActor> sourceWorkers = new ArrayList<>();
+  private List<RayActor> sinkWorkers = new ArrayList<>();
 
   public ExecutionGraph(List<ExecutionNode> executionNodes) {
     this.executionNodeList = executionNodes;
     for (ExecutionNode executionNode : executionNodeList) {
       if (executionNode.getNodeType() == ExecutionNode.NodeType.SOURCE) {
-        List<RayActor<JobWorker>> actors = executionNode.getExecutionTasks().stream()
+        List<RayActor> actors = executionNode.getExecutionTasks().stream()
             .map(ExecutionTask::getWorker).collect(Collectors.toList());
         sourceWorkers.addAll(actors);
       }
       if (executionNode.getNodeType() == ExecutionNode.NodeType.SINK) {
-        List<RayActor<JobWorker>> actors = executionNode.getExecutionTasks().stream()
+        List<RayActor> actors = executionNode.getExecutionTasks().stream()
             .map(ExecutionTask::getWorker).collect(Collectors.toList());
         sinkWorkers.addAll(actors);
       }
     }
     buildTime = System.currentTimeMillis();
   }
 
-  public List<RayActor<JobWorker>> getSourceWorkers() {
+  public List<RayActor> getSourceWorkers() {
```

Comment:
Add some functions like `List<ExecutionNode> getUpstreamExecutionNode(ExeuctionNode node)` and `getDownstreamExecutionNode`

Reply:
These methods doesn't belong to this PR, and they can be put into new execution graph.